app/src/main/java/org/simple/clinic/newentry/PatientEntryUi.kt

```diff
@@ -16,4 +16,5 @@ interface PatientEntryUi {
   fun nextButtonShowInProgress()
   fun enableNextButton()
   fun setupUi(inputFields: InputFields)
+  fun showColonyOrVillagesList(colonyOrVillageList: List<String>)
```

Comment:
Similar comment as before, maybe we start calling it `setColonyOrVillageAutoComplete` or something like that. Let's do this in a different PR